Square_of_Last_Digit_of_nth_Fibonacci_Number/Square_of_Last_Digit_of_nth_Fibonacci_Number.cpp

```diff
@@ -0,0 +1,35 @@
+//C++ Program to find last digit of square of nth fibonacci numbers
+#include<iostream>
+using namespace std;
+
+int main(){
+    int a[60], n, i, c, b[60], j;
+    a[0] = 0;
+    a[1] = 1;
+    b[0] = 0;
+    b[1] = 1;
+    //Using the concept that after 60 numbers in a fibonacci numbers the last digit repeats itself again and again
+    for(i = 2 ; i < 60 ; i++ ){        //for storing the last digits of squares of first 60 fibonacci numbers
```

Comment:
remove the space after i++
can i know why did you include two comment into the same function? it's better if you declare a one comment